README.md

```diff
@@ -0,0 +1,33 @@
+![Mobile Buy SDK](https://cloud.githubusercontent.com/assets/5244861/26738020/885c12ac-479a-11e7-8914-2853ec09f89f.png)
+
+# Storefront App
+
+The sample application demonstrates how to build a custom storefront on iOS using [Shopify's mobile buy SDK](https://github.com/Shopify/mobile-buy-sdk-ios). Using this sample application, a user can browse your shop's products and collections, add merchandise to a cart, and then checkout using Apple Pay or a web checkout.
```

Comment:
@tanderson85 can you review this line in the documentation? It is the only doc change that. Everything else is a port from https://github.com/Shopify/mobile-buy-sdk-ios-sdk

Reply:
@tanderson85 I will merge this in. But please still leave a feedback and I will update